ynh_add_fail2ban_config/ynh_add_fail2ban_config

```diff
@@ -28,21 +28,26 @@ enabled = true
 port = $ports
 filter = $app
 logpath = $logpath
-maxretry = $max_retry" 
+maxretry = $max_retry
 EOF
 
   sudo tee $finalfail2banfilterconf <<EOF
 [INCLUDES]
 before = common.conf
 [Definition]
 failregex = $failregex
-ignoreregrex =" 
+ignoreregrex =
 EOF
 
 	ynh_store_file_checksum "$finalfail2banjailconf"
 	ynh_store_file_checksum "$finalfail2banfilterconf"
   
 	sudo systemctl restart fail2ban
+	if local fail2ban_error="$(tail -n50 /var/log/fail2ban.log | grep "WARNING Command.*$app.*addfailregex")"
+	then
+		echo "[ERR] Fail2ban failed to load the jail for $app" >&2
+		echo "WARNING${fail2ban_error#*WARNING}" >&2
```

Comment:
What is this command supposed to do?

Reply:
Hum, the first line grep an error in the log file.
And, if grep find something, it simply print this part of the log.

Reply:
Hmm thanks 😆 
I was referring to line 49 specifically... 😉

Reply:
Oh ok;)
Sorry that true that I forgot to comment my code! Shame on me...
This print removes the beginning of the line which is essentially a timestamp and other boring stuff